x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateTimeFunctionTestcase.java

```diff
@@ -0,0 +1,72 @@
+package org.elasticsearch.xpack.sql.expression.function.scalar.datetime;
+
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.xpack.sql.expression.function.FunctionContext;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import java.util.Locale;
+import java.util.TimeZone;
+
+public abstract class DateTimeFunctionTestcase<F extends DateTimeFunction> extends ESTestCase {
```

Comment:
nice test case for datetime sql functions to make each test a one liner without any boilerplate.